packages/ember-metal/lib/meta.js

```diff
@@ -67,7 +67,8 @@ const IS_PROXY = 1 << 4;
 if (isEnabled('ember-glimmer-detect-backtracking-rerender') ||
     isEnabled('ember-glimmer-allow-backtracking-rerender')) {
   members.lastRendered = ownMap;
-  members.lastRenderedFrom = ownMap; // FIXME: not used in production, remove me from prod builds
+  members.lastRenderedReferenceMap = ownMap; // FIXME: not used in production, remove me from prod builds
+  members.lastRenderedTemplateMap = ownMap; // FIXME: not used in production, remove me from prod builds
```

Comment:
`runInDebug` doesn't seem to work here. I'd be grateful for any tips on how I can avoid setting these in production build